### PR TITLE
feat(client): translation status banner

### PR DESCRIPTION
**Pull request by scissorsneedfoodtoo**

### CODE REVIEW

**Comment by nhcarrigan** [comment]
Whoops - failing test is my fault. Didn't plan for booleans in the translation objects. Will push a commit to resolve.

**Comment by nhcarrigan** [comment]
test

On smaller screens, do we want the banner to shift down on menu toggle? Or allow the search bar to overlap?

**Comment by scissorsneedfoodtoo** [comment]
Thanks for catching those things @nhcarrigan, especially that bit with the banner on mobile screens. I can't remember how we used to handle it on News back in the day. Do you remember @ahmadabdolsaheb?

Looks a bit off since it's not quite completely covered. May be better to shift it down with the menu toggle, but that's just me.

Not sure if booleans are the best way to go here, but it was just a fast way to get things working. Feel free to make any necessary changes.

**Comment by nhcarrigan** [comment]
Rather than shifting it down (and thus having to shift down the entire page content) what if we shrink it to match the height of the search bar?

test

**Review by raisedadead — COMMENTED** [review]
Let's change the banner message to something like:

```
Translations for these parts are in progress, you can help [here].
```

with the link to:

```
https://contribute.freecodecamp.org/#/how-to-translate-files
```

**Comment by RandellDawson** [comment]
When this PR is merged, we need to run the Crowdin UI upload workflow, so @miyaliu666 and @RafaelDavisH can make the translations.

**Comment by nhcarrigan** [comment]
@scissorsneedfoodtoo if you want to take another look, I think I'm happy with this.

**Comment by scissorsneedfoodtoo** [comment]
@nhcarrigan, that's a great idea. Looks much cleaner that way. I'll make that change now, along with adding a link to the contribute docs.

**Comment by raisedadead** [comment]
Is this ready for review, we can add the translations for the strings after the fact. If yes, a quick screenshot in here, and we can get this signed-off with Quincy before going live.

**Comment by nhcarrigan** [comment]
Sure - give me a moment to spin it up

**Comment by nhcarrigan** [comment]
image

@raisedadead here is how it currently looks. I think @scissorsneedfoodtoo is taking one last look at it.

File: client/i18n/locales/english/intro.json [hunk 1]
```diff
@@ -507,6 +518,8 @@
     "browse-other": "Browse our other free certifications\n(we recommend doing these in order)",
     "courses": "Courses",
     "expand": "Expand",
-    "collapse": "Collapse"
+    "collapse": "Collapse",
+    "translation-banner": "Translations for this section are in progress.",
```
Comment (raisedadead) [inline client/i18n/locales/english/intro.json:522]:
```suggestion
    "translation-banner": "Help us translate these parts of the curriculum,",
```

File: client/i18n/locales/english/intro.json [hunk 2]
```diff
@@ -507,6 +518,8 @@
...
+    "translation-link": "You can help"
```
Comment (raisedadead) [inline client/i18n/locales/english/intro.json:523]:
```suggestion
    "translation-link": "here."
```

**Review by raisedadead — CHANGES_REQUESTED** [review]
Friendlier.

File: client/i18n/intro-schema.js [hunk 3]
```diff
@@ -512,7 +523,9 @@ const introSchema = {
       'Browse our other free certifications\n(we recommend doing these in order)',
     courses: 'Courses',
     expand: 'Expand',
-    collapse: 'Collapse'
+    collapse: 'Collapse',
+    'translation-banner': 'Translation in progress',
```
Comment (raisedadead) [inline client/i18n/intro-schema.js]:
```suggestion
    'translation-banner': "Help us translate these parts of the curriculum,",
```

File: client/i18n/intro-schema.js [hunk 4]
```diff
@@ -512,7 +523,9 @@ const introSchema = {
...
+    'translation-link': 'You can help'
```
Comment (raisedadead) [inline client/i18n/intro-schema.js]:
```suggestion
    'translation-link': "here."
```

**Review by raisedadead — CHANGES_REQUESTED** [review]

**Comment by scissorsneedfoodtoo** [comment]
Sorry all, just seeing your comments now. I had a quick call with Quincy a bit earlier to go over the text and updated it based on his input. Other than that made a few stylistic changes and tried to get the functionality closer to the banner over on News.

**Comment by raisedadead** [comment]
Sure, no worries, just wanted to make sure we have the message a tad friendly. This looks good. Please mark it ready when done and will merge and deploy.

**Comment by scissorsneedfoodtoo** [comment]
Okay, this should be ready for review if anyone wants to take another look.

And here's are some screenshots just in case:

image

image

File: client/i18n/locales/english/intro.json [hunk 5]
```diff
@@ -1,6 +1,7 @@
 {
   "responsive-web-design": {
     "title": "Responsive Web Design",
+    "isTranslated": false,
```
Comment (raisedadead) [inline client/i18n/locales/english/intro.json:4]:
```suggestion
    "isTranslated": true,
```

This should be set to `true`??

Comment (scissorsneedfoodtoo) [reply client/i18n/locales/english/intro.json:4]: Sorry! Forgot to set that back.

**Review by raisedadead — APPROVED** [review]
The broken cypress test is not related to this PR, I am disabling it in a separate PR.

**Pull request merged**